deployment/Deploy.ps1

```diff
@@ -168,6 +207,27 @@ process {
         $UserEmailAddress = $ActiveSubscription.user.name
         #endregion
 
+        Write-Host "Checking if resource group already exists..."
+        $checkResourceGroupExist = (az group exists --resource-group $ResourceGroupName)
+        if($checkResourceGroupExist -eq $true){
+            Write-Host "Resource Group exists. Please cleanup the existing resources to create the new one."
```

Comment:
The first write host implies they need to do this, but then we prompt for it to be done automatically. 

I would change it to something like "The specified Resource Group <<ResourceGroupName>> already exists."
Then
"Would you like to automatically cleanup this resource group, or exit the deployment process?"